test/test_extended_models.py

```diff
@@ -121,6 +121,88 @@ def get_models_from_module(module):
     assert a == b
 
 
+@pytest.mark.parametrize(
+    "module", [models, models.detection, models.quantization, models.segmentation, models.video, models.optical_flow]
+)
+@pytest.mark.parametrize(
+    "include_filters",
+    [
+        None,
+        "*resnet*",
+        ["*alexnet*"],
+        "*not-existing-model-for-test?",
+        ["*resnet*", "*alexnet*"],
+        ["*resnet*", "*alexnet*", "*not-existing-model-for-test?"],
+        ("*resnet*", "*alexnet*"),
+        set(["*resnet*", "*alexnet*"]),
+        {"*resnet*": "test", "*alexnet*": "test"},
+    ],
+)
+@pytest.mark.parametrize(
+    "exclude_filters",
+    [
+        None,
+        "*resnet*",
+        ["*alexnet*"],
+        ["*not-existing-model-for-test?"],
+        ["resnet34", "*not-existing-model-for-test?"],
+        ["resnet34", "*resnet1*"],
+        ("resnet34", "*resnet1*"),
+        set(["resnet34", "*resnet1*"]),
+        {"resnet34": "test", "**resnet1**": "test"},
+    ],
+)
+def test_list_models_filters(module, include_filters, exclude_filters):
+    def get_models_from_module(module):
```

Comment:
I'm not sure we'll keep this but if we do, we might as well extract it out as a stand-alone util to be used in this test and in `test_list_models()` above. 

> I'm not sure we'll keep this

The reason I say that is because it doesn't handle the `quantized_` logic properly and that's a bit unfortunate - we'll see later.